### DISCUSSION TITLE
How to Enable Auto-Completion for Tools Installed via pixi global install?

### DISCUSSION THREAD

**Discussion by kadaliao** [prefix-dev/pixi#2592] [Q&A]
Thank you for creating such an excellent tool! I have a question regarding auto-completion support.

When installing tools like bat using homebrew, they often come with auto-completion scripts. 

![CleanShot 2024-11-29 at 11 55 43](https://github.com/user-attachments/assets/3da67cc2-a7b3-44bb-941c-82b7b0a59d72)

Is there a way to enable or install auto-completion for tools installed using pixi global install?

I would greatly appreciate any guidance or documentation on this.

Thank you in advance for your time and support!

**Comment by wolfv** [comment]
Hi @kadaliao - there is currently no good way, but we have a proposal! https://github.com/prefix-dev/pixi/issues/2366

If you want to work with us, I would be happy to guide you through how we can make a prototype and see if it works!
